site/src/pages/DeploySettingsPage/NotificationsPage/NotificationEvents.tsx

```diff
@@ -0,0 +1,247 @@
+	availableMethods: NotificationMethod[];
+	templatesByGroup: ReturnType<typeof selectTemplatesByGroup>;
+	deploymentValues: DeploymentValues;
+};
+
+export const NotificationEvents: FC<NotificationEventsProps> = ({
+	defaultMethod,
+	availableMethods,
+	templatesByGroup,
+	deploymentValues,
+}) => {
+	// Webhook
+	const hasWebhookNotifications = Object.values(templatesByGroup)
+		.flat()
+		.some((t) => t.method === "webhook");
+	const webhookValues = deploymentValues.notifications?.webhook ?? {};
+	const isWebhookConfigured = requiredFieldsArePresent(webhookValues, [
+		"endpoint",
+	]);
+
+	// SMTP
+	const hasSMTPNotifications = Object.values(templatesByGroup)
+		.flat()
+		.some((t) => t.method === "smtp");
+	const smtpValues = deploymentValues.notifications?.email ?? {};
+	const isSMTPConfigured = requiredFieldsArePresent(smtpValues, [
+		"smarthost",
+		"from",
+		"hello",
+	]);
+
+	return (
+		<Stack spacing={4}>
+			{hasWebhookNotifications && !isWebhookConfigured && (
+				<Alert
+					severity="warning"
+					actions={
+						<Button
+							variant="text"
+							size="small"
+							component="a"
+							target="_blank"
+							rel="noreferrer"
+							href={docs("/admin/notifications#webhook")}
+						>
+							Read the docs
+						</Button>
+					}
+				>
+					Webhook notifications are enabled, but not properly configured.
+				</Alert>
+			)}
+
+			{hasSMTPNotifications && !isSMTPConfigured && (
+				<Alert
+					severity="warning"
+					actions={
+						<Button
+							variant="text"
+							size="small"
+							component="a"
+							target="_blank"
+							rel="noreferrer"
+							href={docs("/admin/notifications#smtp-email")}
+						>
+							Read the docs
+						</Button>
+					}
+				>
+					SMTP notifications are enabled but not properly configured.
+				</Alert>
+			)}
+
+			{Object.entries(templatesByGroup).map(([group, templates]) => (
+				<Card
+					key={group}
+					variant="outlined"
+					css={{ background: "transparent", width: "100%" }}
+				>
+					<List>
+						<ListItem css={styles.listHeader}>
+							<ListItemText css={styles.listItemText} primary={group} />
+						</ListItem>
+
+						{templates.map((tpl, i) => {
+							const value = castNotificationMethod(tpl.method || defaultMethod);
+							const isLastItem = i === templates.length - 1;
+
+							return (
+								<Fragment key={tpl.id}>
+									<ListItem>
+										<ListItemText
+											css={styles.listItemText}
+											primary={tpl.name}
+										/>
+										<MethodToggleGroup
+											templateId={tpl.id}
+											options={availableMethods}
+											value={value}
+										/>
+									</ListItem>
+									{!isLastItem && <Divider />}
+								</Fragment>
+							);
+						})}
+					</List>
+				</Card>
+			))}
+		</Stack>
+	);
+};
+
+function requiredFieldsArePresent(
+	obj: Record<string, string | undefined>,
+	fields: string[],
+): boolean {
+	return fields.every((field) => Boolean(obj[field]));
```

Comment:
Nit: Could use the `in` operator here maybe? `fields.every((field) => field in obj)`

Reply:
We also need to check if the field is not an empty string so I think `Boolean(obj[field])` works best.

Reply:
Ah yeah fair enough 😎